CHANGELOG.md

```diff
@@ -5,31 +5,52 @@ vNext
 ------
 
 (Add your change to a random empty line to avoid merge conflicts)
- -
- -
- - Flax no longer uses implicit rank broadcasting. Thus, you can now use Flax with `--jax_numpy_rank_promotion=raise`.
- -
- -
- -
- - NamedTuples are no longer converted to tuples on assingment to a `linen.Module`.
- -
- -
- - Fixed a bug in FrozenDict which didn't allow copying dicts with reserved names.
- -
- -
- -
- - `flax.linen.enable_named_call` and `flax.linen.disable_named_call` now work anywhere instead of only affecting Modules constructed after the enable/disable call. Additionally, there is now `flax.linen.override_named_call` that provided a context manager to locally disable/enable named_call.
- -
- -
- - Fix the serialization of named tuples. Tuple fields are no longer stored in the state dict and the named tuple class is no longer recreated ([bug](https://github.com/google/flax/issues/1429)).
- -
- - linen Modules and dataclasses made with `flax.struct.dataclass` or `flax.struct.PyTreeNode` are now correctly recognized as dataclasses by static analysis tools like PyLance. Autocomplete of constructors has been verified to work with VSCode.
- -
- -
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+- 
+
+
+
+0.3.5
```

Comment:
Could you group these for easier reading?
(maybe "breaking changes", "new features", and "bugfixes")

Reply:
Done